analysis.py

```diff
@@ -37,14 +40,17 @@ def build_analysis():
   xml_data = {}
   
   for fn in os.listdir('XML'):
-    if 'v30' in fn:
+    if 'swp' not in fn:
```

Comment:
I wonder if we should have a separate function to find correct files - that uses `yield`